packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -82,25 +82,11 @@ const BuySellForm = props => {
             }
 
             advertiser_page_store.setFormErrorMessage('');
-            const disposeRateChangeModal = reaction(
-                () => floating_rate_store.is_market_rate_changed,
-                is_market_rate_changed => {
-                    if (is_market_rate_changed && rate_type === ad_type.FLOAT) {
-                        showModal({
```

Comment:
If we remove this reaction to show the modal when the market rate has changed, the user won't be informed that the market rate has been changed... we only show the modal below here, so I do think we need to keep this

![Screenshot 2023-07-14 at 12 05 59 PM](https://github.com/binary-com/deriv-app/assets/103412909/1d5fc5a8-56d3-4153-98bc-6e07a36d004f)

Reply:
@niloofar-deriv You can trigger the modal in a useEffect with dependency on the exchange rate I think 🤔

Reply:
thanks @ameerul-deriv for mentioning this, let me find a way for it :pray:

Reply:
I'm checking it @farzin-deriv :pray:

Reply:
Done :pray: